packages/gasket-plugin-intl/lib/build-modules.js

```diff
@@ -169,14 +178,54 @@ class BuildModules {
     }, Promise.resolve([]));
   }
 
+  /**
+   * Find modules with locale directories to process
+   *
+   * @returns {SrcPkgDir[]} source package directories
+   */
+  async gatherModuleDirs() {
+    if (this._lookupModuleDirs) {
+      const promises = this._lookupModuleDirs.map(async lookupDir => {
+        const match = lookupDir.match(rePkgParts);
+        const pkgName = match.groups.name;
+        const subDir = (match.groups.dir ?? '/locales').substring(1);
+
+        const buildDir = path.join(
+          this._nodeModulesDir,
+          ...pkgName.split('/'),
+          ...subDir.split('/')
+        );
+
+        let results;
+        try {
+          const stat = await fs.lstat(buildDir);
+          if (stat.isDirectory()) {
+            results = [pkgName, buildDir];
+          }
+        } catch (e) {
+          // skip
+        }
+        if (!results) {
+          this._logger.warning(`build:locales: locales directory not found for: ${lookupDir}`);
+        }
+        return results;
```

Comment:
```suggestion
        try {
          const stat = await fs.lstat(buildDir);
          if (stat.isDirectory()) {
            return [pkgName, buildDir];
          }
        } catch (e) {
          // skip
        }
        this._logger.warning(
          `build:locales: locales directory not found for: ${lookupDir}`
        );
        return null;
```
Quicker return if it is a directory?